src/Sylius/Bundle/TaxonomyBundle/Resources/config/services.xml

```diff
@@ -32,6 +33,10 @@
             <argument type="service" id="sylius.repository.taxon" />
             <tag name="form.type" alias="sylius_taxon_selection" />
         </service>
+
+        <service id="sylius.factory.taxonomy" class="%sylius.factory.taxonomy%">
```

Comment:
Anyway, we don't need any changes in this file, it is handled by `*Extension` and factory definition always gets overwritten.